specification/netapp/resource-manager/Microsoft.NetApp/stable/2019-05-01/netapp.json

```diff
@@ -1696,7 +1696,7 @@
           "type": "array",
           "description": "List of mount targets",
           "items": {
-            "$ref": "#/definitions/mountTargetList"
+            "$ref": "#/definitions/mountTarget"
```

Comment:
this sounds like a Swagger bug fixing, just to confirm. otherwise it will be a breaking change

Reply:
hi @audunn, could you pls confirm this? thanks.

Reply:
Yes, this is a correction mountTargets that is returned is a generic list (array) of mountTargets and is more accureatly represented in swagger like this. This aligns with later API versions of the swagger that included this change in a previous pr.

Reply:
Note we need to make a correction on this the mountTarget property is actually represented like so:
`"$ref": "#/definitions/mountTargetProperties"`